deploy/crds.yml

```diff
@@ -189,6 +189,10 @@ spec:
                   source:
                     type: string
                     description: Messages will be published from that subject to the destination subject.
+              preventDelete:
+                description: Indicates if a Stream can be deleted (true) or not (false)
```

Comment:
This should say `When true, the managed Stream will not be deleted when the resource is deleted`

Reply:
I made this change to the description